### PR TITLE
Fix the task list template to use the correct status tag options

### LINKED ISSUE #27: Task List example template
The example template for the new task-list pattern has an issue causing the pattern to render as pictured.

screenshot showing the task list status tags rendering at varied widths

Looks like the variable created above the macro used to add the status isn't calling the tag component correctly. Currently it's like this:

```
{% set incompleteStatus = {
  text: "Incomplete",
  classes: "govuk-tag--blue"
} %}
```

It works if I change it to this:

```
{% set incompleteStatus = {
  tag: {
    text: "Incomplete",
    classes: "govuk-tag--blue"
  }
} %}
```

### PR DESCRIPTION
**Pull request by frankieroberto**
Fixes #27

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.1
+
+- Fix for a bug with the task list template
```
Comment (colinrotherham) [inline CHANGELOG.md:5]: Shall we quickly explain what's being fixed?

```suggestion
- Fix the task list template to use the correct status tag options
```

**Review by colinrotherham — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.0.1
+
+- Fix the task list template to use the correct status tag options
```
Comment (36degrees) [inline CHANGELOG.md:5]: Can this include the PR number and link, to match the format used in previous versions please?

**Review by owenatgov — DISMISSED** [review]

**Review by 36degrees — COMMENTED** [review]
Normally we'd 'release' (bump the version number) in a separate pull request from the fix.

Would you mind reverting the changes to `package.json`, `package-lock.json` and adding the changelog entry under an 'Unreleased' heading instead? Then we can follow up with another PR when we're ready to publish the changes to npm.

Happy to make those change on your behalf if you prefer.

File: CHANGELOG.md (hunk 2 above)
Comment (frankieroberto) [reply CHANGELOG.md]: Done.

**Review by colinrotherham — APPROVED** [review]

**Pull request merged**